framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -697,6 +698,8 @@ public TestStatus verifyTextPlan(String expectedOutput,
     if (verified) {
       return TestStatus.PASS;
     }
+    actual = actual.substring(0,ALLOWED_ACTUAL_PLAN_VERIFICATION_LENGTH);
```

Comment:
I think we should revert this. We need an intelligent way to crop the text.

Reply:
ok